### PR TITLE
Reformat

### PR DESCRIPTION
**Pull request by noxa617**

### CODE REVIEW

File: init.sqf [hunk 1]
```diff
@@ -4,24 +4,41 @@
 US_VOA = 0; // can elevate all the way to 10
 US_LOW_CT = ["Low Counter Attack", "Mid Counter Attack", "High Counter Attack"];
 
+flagList= ["AgiaMarinaFlag", "CampRogainFlag"];
 townsList = ["AgiaMarina", "CampRogain"];
```
Comment (mattking976) [inline init.sqf:8]: Should have a space here

**Review by mattking976 — COMMENTED** [review]
Left a comment

File: init.sqf (hunk 1 above)
Comment (noxa617) [reply init.sqf:8]: it works go fuck yourself

**Review by mattking976 — APPROVED** [review]
LGTM

**Pull request merged**